src/components/Nav.tsx

```diff
@@ -0,0 +1,43 @@
+type Props = {
+  selected: string;
+  selectTodoFilter: (filter: string) => void;
+};
+
+export const Nav: React.FC<Props> = ({ selected, selectTodoFilter }) => {
+  return (
+    <nav className="filter" data-cy="Filter">
+      <a
+        href="#/"
+        className={
+          selected === 'all' ? 'filter__link selected' : 'filter__link'
+        }
+        data-cy="FilterLinkAll"
+        onClick={() => selectTodoFilter('all')}
```

Comment:
all filters variants ( all, active and other) need to move to constants ( best way to enums)